vic/drivers/image/src/vic_init_output.c

```diff
@@ -175,7 +175,7 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining nlayer dimenension in %s", nc->fname);
     }
 
-    if (options.COORD_DIMS_OUT == 1) {
+    if (global_domain.info.n_coord_dims == 1) {
         status =
             nc_def_dim(nc->nc_id, "lon", nc->ni_size,
```

Comment:
Would we enforce `lon` or is this something that the user can specify (read from config file similar to what we do for forcings)?